hazelcast/src/main/java/com/hazelcast/sql/metrics/MetricNames.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql.metrics;
+
+/**
+ * This class contains names of metrics related to SQL module.
+ * @since 5.3
+ */
+public final class MetricNames {
+
+    /**
+     * Name of SQL module typically used in the MODULE metric tag.
+     */
+    public static final String MODULE_TAG = "sql";
+
+    /**
+     * Total number of queries executed by the SQL engine.
+     */
+    public static final String FAST_QUERIES_EXECUTED = "sql.fast.queries.executed";
```

Comment:
I'd avoid publishing ad-hoc metrics as a part of this PR. We have no SQL metrics yet, it feels weird that we expose the number of "fast" queries, and nothing else, such as the total number of queries. The javadoc is also incorrect.

We should remove this. We use it for tests, but we can rather use a static final field. We can't have private metrics, all metrics are public and must be supported in the future.